Formula/docker-machine-driver-vultr.rb

```diff
@@ -20,6 +20,9 @@ class DockerMachineDriverVultr < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "60610ec215e212a442adf660b34f651db0c97a75cfe7ca908e00c8d450ccd8f4"
   end
 
+  # last commit was in 2017
+  deprecate! date: "2017-08-11", because: :unmaintained
```

Comment:
Let's set this to today. It's a lot harder to work out when to disable this when the deprecation is back-dated.

Reply:
```suggestion
  deprecate! date: "20-07-29", because: :unmaintained
```